### ISSUE TITLE
$file is not located in the allowed paths

### ISSUE THREAD

**Issue by SvenJuergens** [svewap/ws_scss#50]
after update to TYPO3 11.5.1, i got the Error:

' $file  is not located in the allowed paths'  

That always happens when files are included from a site package.
Seems to be related to this change https://github.com/TYPO3/typo3/commit/c86b292962c7d54a66aa9254af5ba00dcbfe6a8f

**Comment by cyMarvin** [comment]
@SvenJuergens I got the same error.
You can resolve this for now by adding the composer packages' extension symlinks to "addAllowedPaths", e.g. by adding the following to your AdditionalConfiguration.php or your ext_localconf.php.

```
$paths = '/typo3conf/ext/site_package/, /typo3conf/ext/other_ext/';
if (!empty($GLOBALS['TYPO3_CONF_VARS']['FE']['addAllowedPaths'])) {
    $GLOBALS['TYPO3_CONF_VARS']['FE']['addAllowedPaths'] .= ', ' . $paths;
} else {
    $GLOBALS['TYPO3_CONF_VARS']['FE']['addAllowedPaths'] = $paths;
}
```

I think this check should be replaced or removed from the extension, since the description for "addAllowedPaths" claims it to be for TypoScript files only.

**Comment by svewap** [comment]
I released a new version 1.2.0 today. This version comes without using FilePathSanitizer which is now possible in TYPO3 11.
